fenix/app/src/main/java/org/mozilla/fenix/gecko/GeckoProvider.kt

```diff
@@ -61,6 +61,7 @@ object GeckoProvider {
             .consoleOutput(context.components.settings.enableGeckoLogs)
             .debugLogging(Config.channel.isDebug || context.components.settings.enableGeckoLogs)
             .aboutConfigEnabled(Config.channel.isBeta || Config.channel.isNightlyOrDebug)
+            .extensionsProcessEnabled(context.components.settings.extensionProcessEnabled)
```

Comment:
I'm bit afraid the we could have issues as this is really early in the initialization process, in line 59 we are `.experimentDelegate(NimbusExperimentDelegate())`. @jhugman even tough we couldn't get this value in the first run, could we get it in substantial runs when nimbus is able to fetch?

Reply:
To provide you more context,  we would like to uplift the extension processor feature to Beta via a [Nimbus rollout](https://mozilla-hub.atlassian.net/browse/WEBEXT-1439) to small % of users to collect early feedback.

Reply:
The `NimbusExperimentDelegate` is the glue between the Nimbus SDK and Gecko. It doesn't need the SDK to be ready before registering it with `GeckoRuntime`.

Nevertheless, Nimbus _is_ already initialized by the time this executes, so you should be good to go here.

Reply:
Awesome thank you for the clarification!!